mobile/ios/WirelessDebug/WirelessDebug/AppDelegate.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  AppDelegate.swift
+//  Wireless Debug iOS Test App
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+    let hostname = NSLocalizedString("hostname", comment: "")
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+        // Start the Wireless Debugger
+        LogStreamer.start(hostname: self.hostname, apiKey: "test")
+
+        NSSetUncaughtExceptionHandler { exception in
```

Comment:
Understood. Be wary in the documentation to remind people that this exists and they have to set it on their own.